src/Command/SearchImportCommand.php

```diff
@@ -46,7 +46,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 $output->writeln(sprintf(
```

Comment:
A Progressbar would be cool for these too :)

Reply:
That would be awesome 🥇